migration-guides/0.15-Migration-Guide.md

```diff
@@ -276,48 +276,405 @@ parcel build index.html --no-source-maps --no-optimize --no-scope-hoist --dist-d
 parcel build index.html --no-source-maps --dist-dir "dist/" # minified bundle for the browser
 ```
 
-## Breaking Changes Made in Core Libraries
+---
 
-### Changes affecting multiple libraries
+## Changes made across all `core`, `contrib`, `web`, and `node` libraries
 
 - Migrated all FFI to ES modules and dropped support for CommonJS modules.
+- Update project and deps to PureScript v0.15.0.
+
+When applicable, the following changes were also made:
+- Removed `MonadZero` type class and all of its deprecated instances.
 - Removed all kind-specific Proxy types (e.g. `SProxy`, `Proxy2`, `Proxy3`, `RLProxy`, etc.)
     - Replace usage of such types with `Type.Proxy (Proxy(..))`.
-- Removed `MonadZero` type class and all of its deprecated instances.
+- Removed `forall proxy. proxy "a symbol" -> ...` workaround.
+
+  This workaround was needed to reduce breakage in `v0.14.0` ecosystem update
+  when polykinded `Proxy` was originally introduced.
+
+## `purescript` Libraries
+
+### purescript-arrays
+
+Breaking changes:
+- Drop deprecated `group'` and `empty` (#219 by @JordanMartinez)
+
+Other improvements:
+- Fixed minor documention issue with `find` (#216 by @JamieBallingall)
```

Comment:
I don’t think we should include small things like this, especially because it isn’t breaking. You can always look at libraries you use to look for stuff like this. To keep the migration guide reasonable in length I think we should prioritize changes that are breaking (you HAVE to change your code) or likely to affect many people (you probably WANT to change your code)